[rabbitmq] Add disk_free_limit param to config

adanin opened the pull request:
PRD-1878
Tested by hands.

bogdando commented on deployment/puppet/rabbitmq/templates/rabbitmq.config:
```diff
@@ -7,7 +7,8 @@
   ,{rabbit, [
-    {tcp_listeners, [<%= port %>]}
+    {tcp_listeners, [<%= port %>]},
+    {disk_free_limit, <%= disk_free_limit %>}
```
It should be {disk_free_limit, [<%= disk_free_limit %>]} , AFAIK

adanin replied:
No. It's a single value parameter. TCP_listeners is a multi value parameter so it is a list.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/333/

mihgen commented:
rebase please. How did you test this?

adanin committed:
[rabbitmq] Add disk_free_limit param to config

mirantis-jenkins commented:
Merged build triggered.

mirantis-jenkins commented:
Merged build started.

adanin commented:
Fixed and rebased. 
Tested by hands. I copied manifests to the master node and made 'puppet apply' on it. Rabbitmq.config became

```
[root@fuel ~]# cat /etc/rabbitmq/rabbitmq.config 
% This file managed by Puppet 2.7.19
% Template Path: rabbitmq/templates/rabbitmq.config
[
  {rabbit, [
    {disk_free_limit, 100000000}
  ]}
  ,{rabbitmq_stomp, [
    {tcp_listeners, [61613]}
  ]}
].
% EOF
```

Rabbitmq-server restarted successfully. Rabbitmqctl output was:

```
[root@fuel ~]# rabbitmqctl status
Status of node rabbit@fuel ...
[{pid,5693},
....................
 {disk_free_limit,100000000},
 {disk_free,17105162240},
....................
```

Please review.

mirantis-jenkins commented:
Merged build finished.

mirantis-jenkins commented:
:+1: Test PASSed.
Refer to this link for build results: http://fuel-jenkins.mirantis.com/job/fuel_syntax_check/376/

mihgen commented:
Please get it reviewed by Fuel Library team, and then we can merge.

mirantis-jenkins commented:
Build triggered.
